name-computer/map.jinja

```diff
@@ -19,6 +19,8 @@
 
 {%- set dns_domain = salt.pillar.get('name-computer:lookup:dns_domain') | default('', true) %}
 {%- set nameserver = salt.pillar.get('name-computer:lookup:nameserver') | default('', true) %}
+{%- set skip_forward_nsupdate = salt.pillar.get('name-computer:lookup:skip_forward_nsupdate') | default('', true) %}
+{%- set skip_reverse_nsupdate = salt.pillar.get('name-computer:lookup:skip_reverse_nsupdate') | default('', true) %}
```

Comment:
These are boolean, can default explicitly to false

```suggestion
{%- set skip_forward_nsupdate = salt.pillar.get('name-computer:lookup:skip_forward_nsupdate') | default(false, true) %}
{%- set skip_reverse_nsupdate = salt.pillar.get('name-computer:lookup:skip_reverse_nsupdate') | default(false, true) %}
```

Reply:
Ok.  Updated.